container/shim/src/modules/registration.js

```diff
@@ -98,9 +98,9 @@ export async function register(initial = false) {
 
   const certBuffer = await fsPromises.readFile(CERT_PATH);
 
-  // Check cert validity on initial registration and at least once daily
-  if (initial || lastInitialRegistration < Date.now() - 24 * 60 * 60 * 1000) {
-    await checkCertValidity(certBuffer, registerOptions);
+  // Check cert validity on initial registration and at least twice daily
```

Comment:
not to be nit picky here but we already have consts in the orch defined for times so might as well use them here.

Reply:
ohh thought this was the orch repo, my bad